templates/Deepcell_Postprocessing.ipynb

```diff
@@ -144,6 +144,15 @@
     "### Afterwards, we can generate expression matrices from the labeling + imaging data"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Returns:\n",
+    "* cell_size_normalized_data: computed by dividing the marker counts in segmentation_labels by their corresponding cell size, however, if the cell size is zero we leave the marker count as is\n",
```

Comment:
This is just an error check thing, we shouldn't have cells of size zero, so I wouldn't include this is the description, makes things confusing